### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by hatcaiday**
https://edu-redmine.framgia.vn/issues/5009

### CODE REVIEW

**Comment by hatcaiday** [comment]
ready

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <div class="col-sm-1">
+          <%= link_to t(".english"), locale:"en" %>
+        </div>
+        <div class="col-sm-1">
+          <%= link_to t(".vietnamese"), locale:"vi" %>
+        </div>
+      </div>
+    </div>
+  </div>
+</footer><!--/Footer-->
```
Comment (mdmahmudurrahman) [inline app/views/layouts/_footer.html.erb]: Please remove all of the comment

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <div class="col-sm-3">
+          <div class="search_box pull-right">
+            <input type="text" placeholder="Search"/>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div><!--/header-bottom-->
+</header><!--/header-->
```
Comment (mdmahmudurrahman) [inline app/views/layouts/_header.html.erb]: Remove all of the comment

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          <div class="logo pull-left">
+            <%= link_to image_tag("home/logo.png", alt: "Shop logo"), root_path %>
+          </div>
+        </div>
+        <div class="col-sm-8">
+          <div class="shop-menu pull-right">
+            <ul class="nav navbar-nav">
+              <li><%= link_to "<i class='fa fa-shopping-cart'></i> Cart".html_safe, "#"%></li>
+              <li><%= link_to t(".user"), "#" %></li>
+              <li><%= link_to "<i class='fa fa-lock'></i> Login".html_safe, "#" %></li>
```
Comment (mdmahmudurrahman) [inline app/views/layouts/_header.html.erb]: Can you try the following
```ruby
<%= link_to "#" do %>
  <i class="fa fa-shopping-cart"></i><%= t ":i18n_key_for_cart" %>
<% end %>
```
And do the same in other places

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
     <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
   </head>
-
   <body>
-    <%= yield %>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert -<%= message_type %>"><%= message %></div>
+      <% end %>
```
Comment (mdmahmudurrahman) [inline app/views/layouts/application.html.erb]: Em, move the flash message to a partial and than render it here.

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                  <%= link_to image_tag("home/product1.png", alt: "") %>
+                  <h2><%= t ".price" %></h2>
+                  <p><%= t ".name_items1" %></p>
+                  <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i><%= t ".add_cart" %></a>
+                </div>
+                <div class="product-overlay">
+                  <div class="overlay-content">
+                    <h2><%= t ".price" %></h2>
+                    <p><%= t ".name_items_canon1" %></p>
+                    <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i><%= t ".add_cart" %></a>
```
Comment (mdmahmudurrahman) [inline app/views/static_pages/home.html.erb]: You can do the same using link_to helper

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                </div>
+              </div>
+              <div class="choose">
+                <ul class="nav nav-pills nav-justified">
+                  <li><a href="#"><i class="fa fa-eye"></i><%= t ".show_detail" %></a></li>
+                </ul>
+              </div>
+            </div>
+          </div>			
+        </div><!--features_items-->
```
Comment (mdmahmudurrahman) [inline app/views/static_pages/home.html.erb]: Remove all of the comments

**Comment by mdmahmudurrahman** [comment]
(commented)

**Comment by hatcaiday** [comment]
ready

File: app/views/shared/_message.html.erb [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<% flash.each do |message_type, message| %>
+  <div class="alert alert - <%message_type%>"><%= message%></div>
```
Comment (mdmahmudurrahman) [inline app/views/shared/_message.html.erb:2]: 1. `<div class="alert alert-<%message_type%>">`
2. `<%= message %>`

**Pull request merged**